packages/cli/package.json

```diff
@@ -27,6 +26,7 @@
     "validate-npm-package-name": "^3.0.0"
   },
   "devDependencies": {
+    "@changesets/types": "^5.2.1",
```

Comment:
where do we depend on this? 🤔

Reply:
https://github.com/Thinkmill/manypkg/blob/4a835ed40fe40b046810b60c00d918a5aa36dfc8/packages/cli/src/npm-tag.ts#L2